src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { Tab } from '../../types/tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: Tab,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  return (
+    <div>
+      <ul>
+        {tabs.map(tab => (
+          <li
+            className={classNames('', {
```

Comment:
redundant blank class
```suggestion
            className={classNames({
```

Reply:
До мене сім'я приїхала, то я останнім тижднем трохи розгублений))
перероблю